src/components/TitleField.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+
+export interface TextProps {
+    textTitle: string,
+    textContent: string,
+}
+
+class TextField extends React.Component<TextProps> {
+    textTitle: string;
```

Comment:
The attributes could have been more concise, since the class name already has `title` in it.

Reply:
you mean `textTitle` to `title`? but this verbose name was intentional so as not to interfere with default HTML attributes.

Reply:
Well one thing or another should be changed then

Reply:
`title` :arrow_right:  `textTitle`